src/legacy/ui/public/vis/map/kibana_map.js

```diff
@@ -650,10 +652,15 @@ export class KibanaMap extends EventEmitter {
 
   _updateDesaturation() {
     const tiles = $('img.leaflet-tile-loaded');
-    if (this._baseLayerIsDesaturated) {
-      tiles.removeClass('filters-off');
-    } else if (!this._baseLayerIsDesaturated) {
+    // Don't apply client-side styling to EMS basemaps
+    if (_.get(this._baseLayerSettings, 'options.origin') === ORIGIN.EMS) {
```

Comment:
Saved object visualizations don't store the `origin` so client-side desaturation is still getting applied on existing maps.